backend/code_review_backend/issues/serializers.py

```diff
@@ -134,8 +138,6 @@ class IssueCheckSerializer(serializers.Serializer):
     analyzer = serializers.CharField()
     check = serializers.CharField()
     total = serializers.IntegerField()
-
-    # TODO: support publishable stats number once we have hash comparison stored
     publishable = serializers.IntegerField(default=0)
```

Comment:
Nit: rename it publishable_count to differentiate it?

Reply:
This serializer is only usable on a statistics endpoint, the backend code should be fine. Are you worried about mis-uses in the frontend ?

Reply:
Yep, worried about misusing because they have the same name. It's really a nit though, feel free to ignore